packages/dds/tree/src/core/tree/anchorSet.ts

```diff
@@ -564,19 +573,49 @@ export class AnchorSet implements ISubscribable<AnchorSetRootEvents> {
 			}
 		};
 
+		const pathVisitors: Map<PathNode, PathVisitor[]> = new Map();
```

Comment:
this could use a comment explaining what the meaning of the keys in this map are, and how the code makes sure to only call path visitors on the appropriate subtrees.

Reply:
Done.